runtime/plaid-stl/src/github/mod.rs

```diff
@@ -151,6 +151,51 @@ pub struct RepositoryCustomProperty {
     pub value: Option<String>,
 }
 
+/// Parameters sent to the runtime when checking a repo's CODEOWNERS file.
+#[derive(Serialize, Deserialize)]
+pub struct CheckCodeownersParams {
+    pub owner: String,
+    pub repo: String,
+}
+
+/// Status for a repo's CODEOWNERS file
+pub enum CodeownersStatus {
+    /// The file is present and has no errors
+    Ok,
+    /// The file is missing
+    Missing,
+    /// The file is present but has at least one error
+    Invalid,
+}
+
+impl From<String> for CodeownersStatus {
+    fn from(value: String) -> Self {
+        match value.as_str() {
+            "Ok" => Self::Ok,
+            "Missing" => Self::Missing,
+            "Invalid" => Self::Invalid,
+            _ => unreachable!(), // we are receiving this string from the runtime, so it should never happen
```

Comment:
This is true but you could use this on strings that didn't come from the runtime accidentally. Thus this should probably be `TryFrom`.